proto/regen/ecocredit/marketplace/v1/events.proto

```diff
@@ -2,8 +2,7 @@ syntax = "proto3";
 
 package regen.ecocredit.marketplace.v1;
 
-option go_package =
-  "github.com/regen-network/regen-ledger/x/ecocredit/marketplace";
```

Comment:
Oof. This is the issue. Looks like `make proto-format` fixes this and protolint doesn't allow it.

Reply:
My bad. We can add an ignore comment or update protolint configuration to allow more characters. Was trying to avoid adding any protolint ignore comments in the proto files.

Maybe something like the following in `.protolint.yaml`:

```
lint:
  rules:
    # ignore max line length conflicting with proto-format
    remove:
      - MAX_LINE_LENGTH
  rules_option:
```

Reply:
ah i just increased the rule to 85 length - should we just remove the rule instead?

Reply:
Either works but removing it will prevent the need to update it later.